WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/AnalyticsTimeRangeCard.swift

```diff
@@ -1,14 +1,34 @@
 import SwiftUI
 
-/// Resuable Time Range card made for the Analytics Hub.
+/// Reusable Time Range card made for the Analytics Hub.
 ///
 struct AnalyticsTimeRangeCard: View {
 
     let timeRangeTitle: String
     let currentRangeDescription: String
     let previousRangeDescription: String
+    @Binding var selectionType: AnalyticsHubViewModel.TimeRangeSelectionType
+
+    @State private var showTimeRangeSelectionView: Bool = false
+
+    init(viewModel: AnalyticsTimeRangeCardViewModel, selectionType: Binding<AnalyticsHubViewModel.TimeRangeSelectionType>) {
+        self.timeRangeTitle = viewModel.selectedRangeTitle
+        self.currentRangeDescription = viewModel.currentRangeSubtitle
+        self.previousRangeDescription = viewModel.previousRangeSubtitle
+        self._selectionType = selectionType
+    }
```

Comment:
Since I had to connect the Binding `selectionType` inside the AnalyticsTimeRangeCard, I removed the init utility declaration from the `AnalyticsTimeRangeCardViewModel` and declared it here. Otherwise, the compiler will throw an error due to the private access of the `_selectionType` reference.